clippy_lints/src/unneeded_struct_pattern.rs

```diff
@@ -32,7 +32,7 @@ declare_clippy_lint! {
     ///     None => 0,
     /// };
     /// ```
-    #[clippy::version = "1.83.0"]
+    #[clippy::version = "1.85.0"]
```

Comment:
Shouldn't this version be `1.86.0`?
```suggestion
    #[clippy::version = "1.86.0"]
```

Reply:
The version of Clippy at the time this PR was merged seems to be 0.1.86:
https://github.com/rust-lang/rust-clippy/blob/34d109fd263a7a22386eeaea0a8c013ab93b9a74/Cargo.toml#L4

Reply:
Yeah, thanks for pointing. But why [we don't have tag for 1.85.0](https://github.com/rust-lang/rust-clippy/tags)?

Reply:
I think `1.85.0` will be added when it become stable.